pages/brand-assets.mdx

```diff
@@ -1,13 +1,52 @@
 ---
 title: Brand assets
-description: Brand assets for Metrom in SVG and PNG format.
+description: Brand Guidelines and assets for Metrom in SVG and PNG format.
 ---
 
 import { MetromSquareLogo } from "../components/assets/logos/metrom-square";
 import { MetromFullLogo } from "../components/assets/logos/metrom-full";
 
-## Brand assets
+## Metrom Brand Guidelines
+
+We’ve created the following guidelines to help others use our brand and assets, including our logo and content consistently. Please contact us at hello@metrom.xyz for any other brand usage not covered by these guidelines.
+
+**Our Logo & Wordmark**: Please do not edit, change, distort, recolor, or reconfigure the METROM logo.
+
+
+### Brand assets
 
 On this page you can download the brand assets for Metrom in SVG and PNG format.
 
-[Download logos pack](https://github.com/metrom-xyz/docs/raw/6506e13aa7dd3da2db1f8cd4caf34401d8f7f81b/public/metrom-logos.zip)
\ No newline at end of file
+[Download logos pack](https://github.com/metrom-xyz/docs/raw/6506e13aa7dd3da2db1f8cd4caf34401d8f7f81b/public/metrom-logos.zip)
+
+### Do's for Metrom Brand Assets
+
+**Representation in Good Faith**
```

Comment:
Let's add `:` on each does, and also on the don'ts below (`**Representation in Good Faith**:`).